airflow-core/tests/unit/observability/traces/test_otel_tracer.py

```diff
@@ -25,11 +25,12 @@
 from opentelemetry.sdk import util
 from opentelemetry.sdk.trace.export.in_memory_span_exporter import InMemorySpanExporter
 
+from airflow._shared.observability.traces.base_tracer import EmptyTrace
+from airflow._shared.observability.traces.otel_tracer import OtelTrace
+from airflow._shared.observability.traces.utils import datetime_to_nano
```

Comment:
```suggestion
from airflow_shared.observability.traces.utils import datetime_to_nano
```

Here, and all others

Reply:
Fixed.

Reply:
That was bad suggestion - how it is is good now though :)